packages/cli/src/runDeploy.ts

```diff
@@ -41,6 +43,10 @@ export const deployOptions = {
     type: "string",
     desc: "The deployment salt to use. Defaults to a random salt.",
   },
+  keyId: {
```

Comment:
can we make this arg more specific like `awsKmsKeyId`?

also can we support this as an env var as well?

```ts
const keyId = opts.awsKmsKeyId ?? process.env.AWS_KMS_KEY_ID;
const account = keyId ? await createKmsAccount(...) : privateKeyToAccount(privateKey);
```